src/containers/treePage/index.tsx

```diff
@@ -196,102 +181,109 @@ const TreePage: React.FC<TreeProps> = ({ siteData, stewardShip, tokens }) => {
-                              <StewardshipForm
-                                onFinish={onFinishRecordStewardship}
-                                form={stewardshipFormInstance}
-                              />
-                            </StewardshipContainer>
-                          </>
-                        ) : (
-                          <>
-                            <Button
-                              type="primary"
-                              size="large"
-                              onClick={onClickAdopt}
-                            >
-                              Adopt
-                            </Button>
-                          </>
-                        )}
-                      </>
-                    ) : (
-                      <>
-                        <Paragraph>Log in to adopt this tree!</Paragraph>
-                        <Button
-                          type="primary"
-                          size={'large'}
-                          onClick={() =>
-                            history.push(Routes.LOGIN, {
-                              destination: location.pathname,
-                            })
-                          }
-                        >
-                          Log In
-                        </Button>
-                      </>
-                    )}
-                  </TreeInfoContainer>
-                </Col>
-                <Col span={7}>
-                  <TreeCareContainer>
-                    <TreeCareTitle>Tree Care Activity</TreeCareTitle>
-                    {stewardShip.map((value: TreeCare, key) => {
-                      return (
-                        <CareEntry key={key}>
-                          <EntryDate>{value.date}</EntryDate>
-                          <Gap />
-                          <EntryMessage>{value.message}</EntryMessage>
-                        </CareEntry>
-                      );
-                    })}
-                  </TreeCareContainer>
-                </Col>
-              </Row>
-            </TreeMainContainer>
-            <Row>
-              <EntrySpace>
-                {latestEntry.map((entry: Entry) => {
-                  return (
-                    <StyledCard key={entry.title}>
-                      <Title level={3}>{entry.title}</Title>
-                      <EntryMessage>{entry.value}</EntryMessage>
-                    </StyledCard>
-                  );
-                })}
-              </EntrySpace>
-            </Row>
-          </TreePageContainer>
-        )}
+                      <MobileTreeCareContainer>
+                        <TreeActivity stewardship={stewardship} />
+                      </MobileTreeCareContainer>
+                    </MobileTreeMainContainer>
+                  )
+                );
+            }
+          })()}
+          <Row>
+            <List
+              dataSource={latestEntry}
+              grid={{
```

Comment:
❓ Why do each of these sizes need to be set?

Reply:
It makes the number of columns shown flexible for screen sizes. I think you could just set the breakpoints, but I preferred to explicitly list each case.